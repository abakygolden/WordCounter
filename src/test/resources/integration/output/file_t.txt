TAKES 1
TARDA 1
TEKSTEN 1
TELLUS 3
TEMPOR 1
TENGAN 1
TENGANL 1
TENIDO 1
TERNURA 1
THAN 1
THAT 1
THEIR 1
THEM 1
THERE 2
THEY 1
THI 1
THICK 1
THIN 1
THROUGH 1
TID 1
TIEMPO 2
TIENE 4
TIL 3
TILLIGE 1
TINCIDUNT 3
TO 8
TODAS 1
TODO 1
TODOS 1
TOLD 1
TOP 4
TORTOR 1
TRABAJAR 1
TREE 1
TRISTIQUE 2
TROUBLE 1
TRUST 3
TSPUBLIKUM 1