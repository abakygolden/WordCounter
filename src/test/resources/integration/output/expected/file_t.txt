TACOS 4
TANDOORI 4
TANGERINE 4
TENDERS 4
TERIYAKI 4
THAI 4
TIRAMISU 4
TOAST 4
TOFU 4
TOMATO 4
TUNA 4